### ISSUE TITLE
Amber fails to create new app on crystal 0.36.1 on Mac

### ISSUE THREAD

**Issue by contentfree** [amberframework/amber#1247]
Fresh install of crystal 0.36.1 from homebrew on macos 10.15.7. Fresh install of amber 0.35.0.

Attempting to create a new app eventually fails during "Installing Dependencies":

```
➜ amber new -d sqlite amber-test
2021-02-05T02:23:01.309819Z   INFO - generate: Rendering App amber-test in /Users/dave/Sites/amber-test
...
2021-02-05T02:23:01.368449Z   INFO - generate: Installing Dependencies
...
Installing ameba (0.12.1)
Postinstall of ameba: make bin && make run_file
Failed postinstall of ameba on make bin && make run_file:
/usr/local/bin/shards build
Dependencies are satisfied
Building: ameba
Error target ameba failed to compile:
Showing last frame. Use --error-trace for full trace.

In /usr/local/Cellar/crystal/0.36.1_2/src/yaml/from_yaml.cr:12:3

 12 | new(YAML::ParseContext.new, parse_yaml(string_or_io))
      ^--
Error: wrong number of arguments for 'Ameba::Rule::Lint::Syntax.new' (given 2, expected 0..1)

Overloads are:
 - Ameba::Rule::Lint::Syntax.new(config = nil)

make: *** [bin/ameba] Error 1
```

Expectation: That a new amber app would be created.
Experienced: It failed to install the dependencies during new app creation.

**Comment by contentfree** [comment]
I did see https://github.com/amberframework/amber/issues/1245 which is also about failing on crystal 0.36 but they actually seemed to get to a different point than I did.

**Comment by contentfree** [comment]
I've created [an issue at the ameba repo](https://github.com/crystal-ameba/ameba/issues/206) as it's likely a problem there and not directly with Amber.

**Comment by factyy** [comment]
@contentfree I have the same problem as well, but decided not to dig into it

**Comment by drujensen** [comment]
Released Amber 0.36.0 to support Crystal 0.36.x
